x/auth/src/keeper.rs

```diff
@@ -217,35 +218,3 @@ fn create_auth_store_key(address: AccAddress) -> Vec<u8> {
 
     prefix
 }
-
-// TODO: so we really need a Module type?
-// pub enum Module {
-//     FeeCollector,
-// }
-
-// impl Module {
-//     pub fn get_address(&self) -> AccAddress {
-//         match self {
-//             Module::FeeCollector => {
-//                 //TODO: construct address from Vec<u8> + make address constant
-//                 //TODO: where do these addresses come from?
-//                 AccAddress::from_bech32("cosmos17xpfvakm2amg962yls6f84z3kell8c5lserqta")
-//                     .expect("hard coded address is valid")
-//             }
-//         }
-//     }
-
-//     pub fn get_name(&self) -> String {
-//         match self {
-//             Module::FeeCollector => "fee_collector".into(),
-//         }
-//     }
-
-//     pub fn get_permissions(&self) -> Vec<String> {
-//         match self {
-//             Module::FeeCollector => vec![],
-//         }
-//     }
-// }
-
-//TODO: copy tests across
```

Comment:
Should keep the TODO about copying tests.

Reply:
A long time ago the auth keeper had tests! https://github.com/rumos-io/gears/blob/3455cee74dde764948e87cd4590c8f885f7e9b92/gears/src/x/auth/auth.rs#L192
That's what `TODO: copy tests across` is referring to.